lib/python-caller-trampoline.py

```diff
@@ -15,7 +15,7 @@
 # Errors are signaled by exiting with non-zero exit code
 # Exceptions are not caught and so will trigger a process exit with non-zero exit code (signaling an error)
 
-import sys, os, json, importlib, copy, base64, io, matplotlib, signal
+import sys, os, json, importlib, copy, base64, io, matplotlib, signal, sklearn, nltk
```

Comment:
I want someone to audit this for impact on trampoline startup time - of course it's free via forking after the initial load, but I'd just rather know what the impact is.

Reply:
I believe that local testing by Binglin showed a +200ms impact on worker startup time.